src/gateways/interfaces/LegacyBeaconGateway.ts

```diff
@@ -0,0 +1,3 @@
+export interface LegacyBeaconGatewayInterface {
```

Comment:
```suggestion
export interface LegacyBeaconGateway {
```

Reply:
And name the implementation for its concrete source, e.g. `BeaconsApiLegacyBeaconGateway`.

See for example the relationship between `EmailServiceGateway` (abstract interface) and `GovNotifyEmailServiceGateway` (concrete implementation).